ui_framework/components/local_nav/_index.scss

```diff
@@ -17,6 +17,12 @@ $localTabTextColor-isSelected: $localNavButtonTextColor-isHover;
 $localBreadcrumbLinkColor: #328caa;
 $localSearchButtonBackgroundColor: #9c9c9c;
 $localSearchButtonTextColor: #ffffff;
+$localCalendarButtonHoverBackgroundColor: $localNavButtonBackgroundColor-isHover;
+$localCalendarCellPrecludedBackgroundColor: rgba(black, 0.05);
+$localCalendarCellSelectedBackgroundColor: #777777;
+$localCalendarCellSelectedTextColor: #ffffff;
+$localCalendarCellHoverBackgroundColor: $globalInfoColor;
```

Comment:
![image](https://d3vv6lp55qjaqc.cloudfront.net/items/3S3p3T0P0M0Z1a3d2R0H/Screen%20Recording%202017-04-24%20at%2011.35%20AM.gif?X-CloudApp-Visitor-Id=59773&v=bf91f058)

The hover state of the cal and the nav within the cal are different. I'd pick one and stick with it.